server/streamlisteners.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const AWSNotifier = require('./modules/aws-notifier.js')
+const notifier = new AWSNotifier()
+const TOPIC = {
+  TopicName: process.env.AUTOMATIC_NOTIFICATIONS_TOPIC_TITLE,
+  TopicArn: null
+}
+
+const checkTopic = function () {
+  return new Promise(function (resolve, reject) {
+    if (TOPIC.TopicArn == null) {
+      notifier.createTopic(TOPIC.TopicName, function (err, data) {
```

Comment:
why we need to create SNS topic if it is already exists ? CloudFormation should create SNS topic